File: api/code-samples/python/eth/transaction/getTransactionByBlockNumberAndIndex.py
import json
from opensdk.sdk import OpenSDK

host = "https://api.baobab.klaytn.net:8651"

blockTag = "0x27"
transactionIndex = "0x0"

sdk = OpenSDK(host)
eth_response = sdk.eth.get_transaction_by_block_number_and_index(blockTag, transactionIndex)

print(json.loads(eth_response.response.data))

File: web3rpc/rpc-specs/code-samples/javascript/klay/transaction/getTransactionByBlockNumberAndIndex.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
const provider = new JsonRpcProvider("https://public-en-baobab.klaytn.net");

    const blockNumber = 118593751
    const transactionIndexPosition = '0x0'

    provider.klay.getTransactionByBlockNumberAndIndex(blockNumber, transactionIndexPosition, {}, (err, data, response) => {
        console.log(data);
    });

}
)()